test/ui-test/src/drivers/SquishDriver.py

```diff
@@ -116,6 +116,10 @@ def click_obj_by_name(objName: str):
     obj = squish.waitForObject(getattr(names, objName))
     squish.mouseClick(obj, squish.Qt.LeftButton)
 
+def click_obj_by_attr(attr: str):
```

Comment:
It's the attribute (json object) that comes from the names map. I use `getattr` to get it (https://www.programiz.com/python-programming/methods/built-in/getattr)
and then I replace the placeholder by the emoji name.
See StatusChatScreen line 266.

To be honest, I didn't know what name to give it, because it's not a name, and it's not an actual Qt object either.